dm/worker/subtask.go

```diff
@@ -462,6 +472,7 @@ func (st *SubTask) Resume() error {
 		st.setStage(pb.Stage_Paused)
 		return err
 	} else if ctx.Err() != nil {
+		st.setStage(pb.Stage_Paused)
```

Comment:
I think this should not be set to `pb.Stage_Paused`, see https://github.com/pingcap/dm/pull/589#discussion_r409342791

Reply:
ok, I remove this and add a comment e34ec49